src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -414,19 +432,27 @@ private void setupHosting(Channel channel, User sender, String hostingId, String
                 return;
             }
 
+            String prUrl = createUploadPermissionPR(channel,sender,issueID,forkTo,users,Collections.singletonList(defaultAssignee));
+            if(StringUtils.isBlank(prUrl)) {
+                out.message("Could not create upload permission pull request");
+            }
+
             // update the issue with information on next steps
             String msg = "At the request of " + sender.getNick() + " on IRC, "
                     + "the code has been forked into the jenkinsci project on GitHub as "
                     + "https://github.com/jenkinsci/" + forkTo
                     + "\n\nA JIRA component named " + forkTo + " has also been created with "
                     + defaultAssignee + " as the default assignee for issues."
-                    + "\n\nPlease remove your original repository so that the jenkinsci repository "
+                    + "\n\nA [pull request|" + prUrl +"] has been created against the repository permissions updater to "
+                    + "setup release permissions for " + defaultAssignee + ". Additional users can be added by modifying "
+                    + "the created file. " + defaultAssignee + " will need to login to Jenkins' [Artifactory|https://repo.jenkins-ci.org/webapp/#/login] once before the permissions will be merged."
```

Comment:
I thought this was displayed to the user through GitHub.  If so, don't the hyperlinks need to GitHub markdown rather than Confluence style hyperlinks?

Reply:
This is added to the Jira issue when it is closed, so it needs to use Jira syntax.